kolena/_experimental/object_detection/evaluator_multiclass.py

```diff
@@ -351,40 +336,7 @@ def compute_test_case_plots(
         configuration: Optional[ThresholdConfiguration] = None,
     ) -> Optional[List[Plot]]:
         assert configuration is not None, "must specify configuration"
-        thresholds = self.get_confidence_thresholds(configuration)
-        all_bbox_matches = self.matchings_by_test_case[configuration.display_name()][test_case.name]
-
-        # clean matching for confusion matrix
-        match_matched = [
-            (gt, inf) for match in all_bbox_matches for gt, inf in match.matched if inf.score >= thresholds[inf.label]
-        ]
-        match_unmatched_gt = [
-            (gt, inf)
-            for match in all_bbox_matches
-            for gt, inf in match.unmatched_gt
-            if inf is not None and inf.score >= thresholds[inf.label]
-        ]
-        confusion_matrix_matchings = [
-            MulticlassInferenceMatches(
-                matched=match_matched,
-                unmatched_gt=match_unmatched_gt,
-                unmatched_inf=[],
-            ),
-        ]
-
-        plots: Optional[List[Plot]] = []
-        plots.extend(
-            filter(
-                None,
-                [
-                    compute_f1_plot_multiclass(all_bbox_matches),
-                    compute_pr_plot_multiclass(all_bbox_matches),
-                    compute_confusion_matrix_plot(confusion_matrix_matchings),
```

Comment:
I ripped out stuff like the confusion matrix because it also heavily depends on a threshold being assumed for multiple classes.